### DISCUSSION TITLE
server-down/

### DISCUSSION THREAD

**Discussion by giscus** [zmike/zmike.github.io#25] [Announcements]
# server-down/

gitlab is down, post low-effort blogs and touch grass until it returns

https://www.supergoodcode.com/server-down/

**Comment by ermarch** [comment]
I made it look like I did something today.
It worked moderately well.

**Reply by zmike** [reply]
great work

**Comment by BasiqueEvangelist** [comment]
Right when I wanted to look at wlroots :sob:

**Reply by zmike** [reply]
don't bother

**Comment by jams3223** [comment]
Whyyyy!!! 😭😭

**Reply by zmike** [reply]
because servers need naps too

**Comment by n8pjl** [comment]
I think that it's back up!

**Reply by zmike** [reply]
seems like it
